src/kernel/arch/x86_64/mem/vmm_mapping.c

```diff
@@ -47,7 +47,7 @@ void *map_phys_to_virt_addr_hh(void* physical_address, void* address, size_t fla
             clean_new_table(new_table_hhdm);
             pdpr_root = new_table_hhdm;
         } else {
-            pretty_log(Verbose, "No need to allocate pml4");
+            //pretty_log(Verbose, "No need to allocate pml4");
```

Comment:
instead of commenting these out, why not remove them? can always re-add them later if needed